src/index/CompressedBlockPrefiltering.h

```diff
@@ -0,0 +1,140 @@
+//  Copyright 2024, University of Freiburg,
+//                  Chair of Algorithms and Data Structures
+//  Author: Hannes Baumann <baumannh@informatik.uni-freiburg.de>
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "global/Id.h"
+#include "global/ValueIdComparators.h"
+#include "index/CompressedRelation.h"
+
+// The compressed block metadata (see `CompressedRelation.h`) that we use to
+// filter out the non-relevant blocks by checking their content of
+// `firstTriple_` and `lastTriple_` (`PermutedTriple`)
+using BlockMetadata = CompressedBlockMetadata;
```

Comment:
Please move this alias into the `prefilterExpressions` namespace to not clutter the global namespace.